ui/src/dashboards/components/variablesControlBar/VariableDropdown.tsx

```diff
@@ -37,47 +37,54 @@ type Props = StateProps & DispatchProps & OwnProps
 
 class VariableDropdown extends PureComponent<Props> {
   render() {
-    const {selectedValue} = this.props
+    const {selectedKey} = this.props
     const dropdownValues = this.props.values || []
 
     return (
       <div className="variable-dropdown">
         {/* TODO: Add variable description to title attribute when it is ready */}
         <Dropdown
-          selectedID={selectedValue}
+          selectedID={selectedKey}
           onChange={this.handleSelect}
           widthPixels={140}
           titleText="No Values"
           customClass="variable-dropdown--dropdown"
           menuColor={DropdownMenuColors.Amethyst}
         >
-          {dropdownValues.map(v => (
-            <Dropdown.Item key={v} id={v} value={v}>
-              {v}
+          {dropdownValues.map(([key]) => (
+            /*
+              Use key as value since they are unique otherwise 
```

Comment:
If I understand this, it is handling the case where there are multiple items in a map variable with the same key, right? Like:
```
foo,123
foo,456
```
In this case, shouldn't `foo` appear twice in the dropdown? Not sure.

Reply:
Oh nevermind, I think I misunderstood. Disregard.